### PR TITLE
[BACKPORT] DOC-12568: Copy fix

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Backport PR for the customer found issue: https://jira.issues.couchbase.com/browse/DOC-12568

Fly-by grammar fix also.

### CODE REVIEW

File: modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc [hunk 1]
```diff
@@ -435,7 +435,7 @@ By default, when a user loses access to a channel, the next Couchbase Lite Pull
 
 Documents that exist in multiple channels belonging to the user (even if they are not actively replicating that channel) are not auto-purged unless the user loses access to all channels.
 
-Users will be receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation; this is sent regardless of the current auto-purge setting.
+Users will receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation when the auto-purge setting is set to `true`.
```
Comment (simon-dew) [inline modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc]: There's an ambiguity here: is it

1. "Users will receive (x if y) when z", or
2. "Users will receive x if (y when z)"

Do you see what I mean? I'm guessing it's the first one. To avoid this ambiguity, I would rephrase as

"When z, users will receive x if y."

```suggestion
When the auto-purge setting is set to `true`, users will receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation.
```

**Review by simon-dew — DISMISSED** [review]
One minor rephrase for clarity

**Review by simon-dew — APPROVED** [review]
LGTM 👍 thanks!

**Comment by pasin** [comment]
@velicuvlad and @borrrden Can you look at the PR validation script? It seems to download the wrong binary.

**Review by pasin — APPROVED** [review]

**Comment by borrrden** [comment]
The PR validation is meant to validate against in progress versions.  The jobs should be changed to run against 3.2.1 instead I think.

**Comment by velicuvlad** [comment]
> The PR validation is meant to validate against in progress versions. The jobs should be changed to run against 3.2.1 instead I think.

That is correct.

I have an update for the iOS script anyway and will be contained in the upcoming PR that will address some snippet issues, so I'll fix the validation then, as it is not needed for this change.

**Review by velicuvlad — APPROVED** [review]

**Review by borrrden — APPROVED** [review]

**Comment by ElliotFrancisHunter** [comment]
@borrrden @velicuvlad thanks for checking. I've now merged.
